include/inertial_sense_ros.h

```diff
@@ -377,4 +369,5 @@ class InertialSenseROS //: SerialListener
     float magDeclination_ = 0;
     int insDynModel_ = INS_DYN_MODEL_AIRBORNE_4G;
     bool refLLA_known = false;
+    int ioConfig_ = 39624800; //F9P RUG2 RTK CMP: 0x025ca060
```

Comment:
Why don't we just use the hexidecimal value (0x025ca060) here rather than the integer value?  It's easier to read and parse.